Subtester/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -0,0 +1,73 @@
+// GENERATED BY UNITY. REMOVE THIS COMMENT TO PREVENT OVERWRITING WHEN EXPORTING AGAIN
+
+// Android Resolver Repos Start
+([rootProject] + (rootProject.subprojects as List)).each {
+    ext {
+        it.setProperty("android.useAndroidX", true)
+        it.setProperty("android.enableJetifier", true)
+    }
+}
+([rootProject] + (rootProject.subprojects as List)).each { project ->
+    project.repositories {
+        def unityProjectPath = $/file:///**DIR_UNITYPROJECT**/$.replace("\\", "/")
+        maven {
+            url "https://maven.google.com"
+        }
+        mavenLocal()
+        jcenter()
+        mavenCentral()
+    }
+}
+// Android Resolver Repos End
+apply plugin: 'com.android.library'
+**APPLY_PLUGINS**
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+// Android Resolver Dependencies Start
+    implementation 'androidx.annotation:annotation:[1.2.0]' // Assets/RevenueCat/Plugins/Editor/RevenueCatDependencies.xml:5
+    implementation 'com.revenuecat.purchases:purchases-hybrid-common:[1.6.2]' // Assets/RevenueCat/Plugins/Editor/RevenueCatDependencies.xml:4
```

Comment:
Previously, ED4MU would download all aars from the all transitive dependencies and include each of them as `implementation`. This way (with the mainTemplate.gradle) ED4MU adds the dependencies here now instead of downloading all the transitive aars. 

This is more intuitive and easier to work with than having to deal with all the aars. Although, I think we could use includeBuild with both ways, I think I prefer this more.

This is only part of our Subtester project and not part of the plugin that's released.